arrows/core/transform_detected_object_set.cxx

```diff
@@ -182,8 +182,8 @@ backproject_bbox( const kwiver::vital::camera_perspective_sptr camera,
 
   vector_4d back_plane;
   back_plane << n, d;
-  vector_3d p5 =
-    this->backproject_to_plane( camera, vector_2d(box(0), box(1)), back_plane );
+  vector_3d p5 = this->backproject_to_plane(
+    camera, vector_2d(bbox.min_x(), bbox.min_y()), back_plane );
```

Comment:
not critical, but `bbox.upper_left()` is the same as `vector_2d(bbox.min_x(), bbox.min_y())`